firstradeAPI.py

```diff
@@ -41,10 +41,11 @@ def firstrade_init(FIRSTRADE_EXTERNAL=None):
             print(f"The following Firstrade accounts were found: {account_list}")
             print("Logged in to Firstrade!")
             firstrade_obj.set_logged_in_object(name, firstrade)
-            for i, account in enumerate(account_list):
-                firstrade_obj.set_account_number(name, account)
+            for i, entry in enumerate(account_info.all_accounts):
```

Comment:
Not sure why this changed, according to Line 40 `account_list = account_info.all_accounts`

Reply:
account_list = account_info.account_numbers just makes a list and is a different attribute
line 40 is redundant now and can be just removed and account_info.account_numbers in the print where "account_list" is
```suggestion
            for i, entry in enumerate(account_info.all_accounts):
```